src/java.base/share/classes/java/util/ArrayList.java

```diff
@@ -40,11 +40,12 @@
  * {@code Vector}, except that it is unsynchronized.)
  *
  * <p>The {@code size}, {@code isEmpty}, {@code get}, {@code set},
- * {@code iterator}, and {@code listIterator} operations run in constant
- * time.  The {@code add} operation runs in <i>amortized constant time</i>,
- * that is, adding n elements requires O(n) time.  All of the other operations
- * run in linear time (roughly speaking).  The constant factor is low compared
- * to that for the {@code LinkedList} implementation.
+ * {@code getFirst}, {@code getLast}, {@code removeLast}, {@code iterator},
+ * {@code listIterator}, and {@code reversed} operations run in constant time.
+ * The {@code add}, and {@code addLast} operations runs in <i>amortized
+ * constant time</i>, that is, adding n elements requires O(n) time.  All of
```

Comment:
Should `n` and `O(n)` here be in italics or code font?

Reply:
It wasn't in italic or code font before so I didn't change that.

Reply:
Fine to leave it as is in that case.